evap/staff/templates/staff_user_form.html

```diff
@@ -32,6 +32,16 @@ <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create u
         {% endif %}
     </div>
 
+    {% if user_with_same_email %}
+        <div class="alert alert-warning alert-dismissible">
+            <p>
+                {% trans "There is already another user with this email address. You probably want to merge the users." %}
+            </p>
+            <a type="button" class="btn btn-primary btn-sm" href="{% url 'staff:user_merge' form.instance.id user_with_same_email.id %}">{% trans "Merge both users" %}</a>
```

Comment:
```suggestion
            <a type="button" class="btn btn-primary btn-sm" href="{% url 'staff:user_merge' user_with_same_email.id form.instance.id %}">{% trans "Merge both users" %}</a>
```

In the specification, I wrote it the wrong way round - please make the existing user the main user of the merge.